qiskit/compiler/transpiler.py

```diff
@@ -666,8 +666,14 @@ def _parse_basis_gates(basis_gates, backend, circuits):
 def _parse_inst_map(inst_map, backend, num_circuits):
     # try getting inst_map from user, else backend
     if inst_map is None:
-        if hasattr(backend, "defaults"):
-            inst_map = getattr(backend.defaults(), "instruction_schedule_map", None)
+        backend_version = getattr(backend, "version", 0)
+        if not isinstance(backend_version, int):
+            backend_version = 0
+        if backend_version <= 1:
+            if hasattr(backend, "defaults"):
+                inst_map = getattr(backend.defaults(), "instruction_schedule_map", None)
+        else:
+            inst_map = backend.target.instruction_schedule_map()
```

Comment:
This is a good catch, do you mind spinning this off into a separate PR with a dedicated test? I ask because having this as a standalone fix is good for potential backport (not that we'll have a 0.19.3 release with 0.20.0 pending). But also it will leave this as a single logical change and we can concentrate on just the fake providers for this PR.

Reply:
Sure. I caught this while doing some local tests in a jupyter notebook.

Reply:
This part is now removed from this PR.